interface/backend/api/views.py

```diff
@@ -22,22 +22,27 @@
 from ..cases import enums
 
 
-class CaseViewSet(viewsets.ModelViewSet):
+class ViewSetBase(viewsets.ModelViewSet):
```

Comment:
Request this get a more descriptive name like `RequestNoneViewSet`.

Reply:
Not sure about this. This class may contain other basic functions in future, so we can call it `ViewSetBase` and put all basic configuration here without fixing all references of it. What do you think?

Reply:
Fair enough, I think it's fine for now.